creds

```diff
@@ -0,0 +1,2 @@
+USER="test_user"
```

Comment:
but we may not need to commit it right?

Reply:
No, that's not needed, it's an example file

Reply:
hmm but we can document the structure of creds file.

anyway i have left another note which may change the way we pass creds.